BoilerPlates/react-typescript/README.md

```diff
@@ -0,0 +1,40 @@
+# SciChart Wrapper with React and Typescript
+
+This example provides a template for an application using scichart with React and Typescript.
```

Comment:
Add something here to say this is now all handled by scichart-react and to use that unless you need even more low level control.